.gitignore

```diff
@@ -8,3 +8,4 @@
 /Release/
 /Examples/bin/
 *.tmp_proj
+src/.idea/*
```

Comment:
The asterisk isn't needed, see the other lines

Reply:
`.idea/` is probably all that's needed since it seems like that thing could be added to any dir